drivers/wdm/audio/backpln/portcls/private.hpp

```diff
@@ -7,19 +7,24 @@
 #ifndef PORTCLS_PRIVATE_H
 #define PORTCLS_PRIVATE_H
 
-#include <stdio.h>
-
 //#define _KS_NO_ANONYMOUS_STRUCTURES_
 #define PC_IMPLEMENTATION
 #define COM_STDMETHOD_CAN_THROW
 #define PC_NO_IMPORTS
 
 #include <ntddk.h>
 #include <portcls.h>
+#define NDEBUG
+#include <debug.h>
+
 #include <dmusicks.h>
 #include <kcom.h>
-
 #include "interfaces.hpp"
+#include <ks.h>
+#include <ksmedia.h>
+#include <stdio.h>
+
+//#include <intrin.h>
```

Comment:
Skip that commented include. It is not needed obviously.

Reply:
Fixed.